internal/cmd/mongodbflex/instance/update/update.go

```diff
@@ -86,7 +86,7 @@ func NewCmd() *cobra.Command {
 			}
 
 			if !model.AssumeYes {
-				prompt := fmt.Sprintf("Are you sure you want to update instance %s?", instanceLabel)
+				prompt := fmt.Sprintf("Are you sure you want to update instance %s? (This may cause downtime)", instanceLabel)
```

Comment:
Should we confirm with the team? Maybe for the MongoDBFlex this is not the case

Reply:
Removed, will be added later if it applies